### ISSUE TITLE
Possible UTC keystore modification bug

### ISSUE THREAD

**Issue by asyms** [ethereum/go-ethereum#15633]
Hi all,
please don't shoot me if this doesn't belong here, but as my stackexchange post isn't getting any real attention I thought some devs here might have a better insight.

The issue is more extensively explained here: https://ethereum.stackexchange.com/questions/32367/incorrect-passworddid-geth-possibly-modify-my-utc-keystore

In short:

I tried to do a transaction from my etherbase account and was prompted for my password. It said my password was wrong but I'm pretty sure it's one of my regularly used passwords..
I've tried:
- typo's
- directly in geth console with (geth account update ...) or (personal.unlockAccount(....))
- importing in MEW
- CAPS Lock on, NUM Lock off

After trying all of these, I started looking more in detail at the UTC Keystore and noticed the file name has the format: UTC--2017--06--04T.... So the file was created on the 4th of June 2017.
But Windows says the last modification date is the 19th of June 2017.

I don't specifically recall changing my password, but wouldn't Geth change the file name when modifying the password? I really doubt I changed the password.. 

If anybody has any useful information that might help, I'd definitely reward you!

Kind regards

#### System information

Geth version: 1.7.3
OS & Version: Windows

#### Expected behaviour

Account unlocks after password is entered

#### Actual behaviour

Account doesn't unlock

#### Steps to reproduce the behaviour

/

**Comment by RichHorrocks** [comment]
Hi there. I don't really have an answer, but a suggestion hat might be worth trying. I had an issue in the distant past where the keyboard on my laptop had for some reason switched from UK to US layout (and back again, I think). My password contained a special character that was in a different position on these 2 layouts, meaning I was attempting to enter the password containing whatever was in that position of a US keyboard, which was wrong. Perhaps check if the passwords you're trying have such characters, and try another-layout-version of them in the appropriate place. Others may have better/more correct suggestions. Good luck.

**Comment by asyms** [comment]
> RichHorrocks commented
Hi there. I don't really have an answer, but a suggestion hat might be worth trying. I had an issue in the distant past where the keyboard on my laptop had for some reason switched from UK to US layout (and back again, I think). My password contained a special character that was in a different position on these 2 layouts, meaning I was attempting to enter the password containing whatever was in that position of a US keyboard, which was wrong. Perhaps check if the passwords you're trying have such characters, and try another-layout-version of them in the appropriate place. Others may have better/more correct suggestions. Good luck.

Hi there,

Thanks for your response!
Unfortunately I've also wrote scripts for different keyboard layouts, which were all unsuccessful.

At this point, I've tried so many "typo's" that I really doubt that the password is wrong, I'm leaning more towards the "something went wrong with the creation" side.. I don't mean this in any disrespectful way to the developpers.. 

Thanks in advance
